### ISSUE TITLE
Feature: 

### ISSUE THREAD

**Issue by Codehagen** [Codehagen/Dingify#60]
### Type of feature
💡 Feature

### Current behavior
The UserChartActivity component currently displays static data for events and users over time.
![Screenshot 2024-06-11](https://github.com/Codehagen/Dingify/assets/24507211/1379e077-4dd6-4ac1-a555-fc3b7a5b434c)

### Suggested solution
Update the UserChartActivity component to dynamically display information about the events a user has made. The parent component already has the actions integrated. The updated component should:

- Fetch User Data: Utilize the existing data fetching mechanisms in the parent component to retrieve user-specific event data.
- Update Data Format: Ensure the data format aligns with what the LineChart component expects.
- Modify the Tooltip: Adjust the tooltip to display user-specific event information.

**Comment by Motormary** [comment]
This chart + activity calendar is redundant because they will display the same data. It should probably be moved to the dashboard.

**Comment by Codehagen** [comment]
Thats true, do we have any other cool charts or query info we can use? 🤔✨

We talked about the user jounery, maybe have a [Scatter Chart](https://recharts.org/en-US/examples/SimpleScatterChart) to display how "interested" a user is?
